UNRELEASED.md

```diff
@@ -0,0 +1,4 @@
+**Added**
+
+- bpk-component-page-indicator
+  - The page indicator is a new component that will be used to display multiple swappable images.
```

Comment:
Sorry, you have to update it again since this component doesn't have swappable functionality

Reply:
Have updated a new instrument, hope to be reviewed again! Thank you!